### ISSUE TITLE
TextRoom didn't receive message from other users

### ISSUE THREAD

**Issue by mohanasundaramiosdev** [crossle/janus-gateway-ios#10]
Hi,
I am using your code for text room chat message. I can able to send a message from my iOS app. But, I can't able to receive any messages from other users in the same room. I spent around 2 weeks, But I am not able to find any solution for it.  Kindly request you to give me solution.
Thanks

**Comment by crossle** [comment]
This only support video room

**Comment by mohanasundaramiosdev** [comment]
Yes, I saw that. But we are able to send and receive message in Android app. 
For this, we just replaced plugin like this  @"plugin": @"janus.plugin.textroom". 

So is it possible to do this in iOS ?

Thanks

**Comment by crossle** [comment]
Sorry，I not use text room, plz test youself

**Comment by mohanasundaramiosdev** [comment]
Okay Crossle,
Thanks much for your response.
